src/main/java/com/alevel/project/coffee/service/UserServiceImpl.java

```diff
@@ -0,0 +1,43 @@
+package com.alevel.project.coffee.service;
+
+import com.alevel.project.coffee.model.User;
+import com.alevel.project.coffee.model.enums.Role;
+import com.alevel.project.coffee.repository.UserRepo;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Service;
+
+import java.util.Collections;
+
+import static com.alevel.project.coffee.model.enums.Status.ACTIVE;
+
+@Service
+public class UserServiceImpl {
```

Comment:
what implement?

Reply:
at the branch re-captcha was added UserService interface and updated service layer